core/src/test/java/com/linecorp/armeria/server/http/dynamic/DynamicHttpServiceBuilderTest.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2017 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linecorp.armeria.server.http.dynamic;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import org.junit.Test;
+
+import com.linecorp.armeria.common.RequestContext;
+import com.linecorp.armeria.common.http.AggregatedHttpMessage;
+import com.linecorp.armeria.common.http.HttpRequest;
+
+public class DynamicHttpServiceBuilderTest {
+
+    @Test
+    public void testBuildingAnnotatedService() {
```

Comment:
How about splitting this method into two test methods (one that tests the successful case and the other that tests the failing case)?